App.js

```diff
@@ -1,26 +1,68 @@
-import React from 'react';
-import styled from 'styled-components/native';
+import React from 'react'
+import { Text, ImageBackground, StyleSheet, View } from 'react-native'
+import ShakeAnswer from './components/ShakeAnswer'
 
-const Container = styled.View`
-	flex: 1;
-	background-color: papayawhip;
-	justify-content: center;
-	align-items: center;
-`;
 
-const Title = styled.Text`
-	font-size: 24px;
-	color: palevioletred;
-`;
+const styles = StyleSheet.create({
+	container: {
+	  flex: 1,
+	  justifyContent: "center",
+	  alignItems: 'center',
+	  backgroundColor: 'black'
+	},
+	title: {
+		color: 'white',
+		paddingBottom: 100,
+		fontSize: 40,
+		fontFamily: "monospace"
+	},
+	image: {
+	  flex: 1,
+	  justifyContent: "center",
+	  width: '100%',
+	  borderBottomLeftRadius: 300,
+	  borderBottomRightRadius: 300,
+	  borderTopLeftRadius: 300,
+	  borderTopRightRadius: 300,
+	  overflow: 'hidden',
+	},
+	text: {
+	  color: "white",
+	  fontSize: 25,
+	  textAlign: "center",
+	  fontWeight: "bold",
+	  fontFamily: "monospace"
+	},
+	SmallText: {
+		color: 'white',
+		textAlign: "center",
+		fontFamily: "monospace"
+	}, SmallViewSecond: {
+		flex: 1,
+		alignItems: 'center',
+		justifyContent: "center",
+	},
+	SmallView: {
+		height: 20,
+	}
+  })
 
 const App = () => {
 	return (
-		<Container>
-			<Title>This is your cool app!</Title>
-			<Title>Go to App.js and start coding</Title>
-			<Title>💅💅💅</Title>
-		</Container>
+		<View style={styles.container}>
+			<Text style={styles.title}>MAGIC 8-BALL</Text>
+			<View style= {{width: '100%', height: '50%',  alignItems: 'center'}}>
```

Comment:
Perhaps this styling could be also in the stylesheet when you already have one made?